Homework13/PageObjects/WebTablePage.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Text;
+using Homework13.Common.WebElements;
+using OpenQA.Selenium;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace Homework13.PageObjects
+{
+    public class WebTablePage
+    {
+        private DemoQaWebElement _addNewEntryButton = new(By.XPath("//button[@id='addNewRecordButton']"));
+        private DemoQaWebElement _submitButton = new(By.XPath("//button[@id='submit']"));
+        private DemoQaWebElement _form = new(By.XPath("//form"));
+        private DemoQaWebElement _firstName1 = new(By.XPath("((//div[@role='rowgroup'])[1]//descendant::div[@role='gridcell'])[1]"));
```

Comment:
Bad naming. Try to create method which takes column name and row number and return the result